.github/workflows/aws-deploy.yml

```diff
@@ -18,8 +18,20 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
-      - name: Static Analysis
-        uses: pre-commit/action@v3.0.0
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
```

Comment:
if you want to keep everything as-is i suggest you simply add a `setup-python` action.. 

```
      - uses: actions/setup-python@v5
        with:
          python-version: "3.x"
```

Reply:
Thanks. The solution that you proposed worked. I will keep everything as is then. Thank you Khai!